cmake-tool/helpers/make_cpio.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
```

Comment:
This introduces a dependency on libarchive. Is this dependency encoded somewhere?

Reply:
I assume it's the same as the dependency here: https://github.com/seL4/seL4/blob/master/tools/python-deps/setup.py#L39